api_server/src/lib.rs

```diff
@@ -44,7 +44,25 @@ use vmm::VmmAction;
 #[derive(Debug)]
 pub enum Error {
     Io(io::Error),
-    Eventfd(sys_util::Error),
+    Eventfd(io::Error),
+}
+
+impl fmt::Display for Error {
```

Comment:
You will also need to change the following code in src/main.rs
```rust
    match server.bind_and_run(
        uds_path_or_fd,
        start_time_us,
        start_time_cpu_us,
        seccomp_level,
    ) {
        Ok(_) => (),
        Err(Error::Io(inner)) => match inner.kind() {
            ErrorKind::AddrInUse => panic!(
                "Failed to open the API socket: IO Error: {:?}",
                Error::Io(inner)
            ),
            _ => panic!(
                "Failed to communicate with the API socket: IO Error: {:?}",
                Error::Io(inner)
            ),
        },
        Err(Error::Eventfd(inner)) => panic!(
            "Failed to open the API socket: EventFd Error: {:?}",
            Error::Eventfd(inner)
        ),
    }
```
because we will end up with errors like:
```bash
Failed to communicate with the API socket: IO Error: IO Error: address already in use.
```
I would change all these to `Failed to communicate with the API socker. {:?}`

Reply:
Done